db/migrate/20221123162401_change_other_known_claimants_to_be_radio_input.rb

```diff
@@ -0,0 +1,17 @@
+class ChangeOtherKnownClaimantsToBeRadioInput < ActiveRecord::Migration[7.0]
+  class Claim < ActiveRecord::Base
+    self.table_name = :claims
+  end
+
+  def up
+    Claim.all.each do |claim|
+      claim.update_columns other_known_claimants: claim.other_known_claimants
+    end
+  end
+
+  def down
+    Claim.all.each do |claim|
```

Comment:
I can't see a migration that adds the other_known_claimants column ?  Should that not be in this issue or am I missing something.  

This down block seems wrong anyway - but, it is probably not possible to do a down without knowing what the previous value was - unless you know it was nil of course ?